instruments/tests/test_thorlabs/test_thorlabs_apt.py

```diff
@@ -1174,6 +1174,74 @@ def test_apt_mc_position(init_kdc101):
         )
 
 
+def test_apt_mc_backlash_correction(init_kdc101):
```

Comment:
Can we split this up into a few different tests for readability

Reply:
Split up into three tests: unitless & counts / unitful / bad units